client/mds3/legend.js

```diff
@@ -824,37 +822,25 @@ function may_create_cnv(tk, block) {
 function may_update_cnv(tk) {
 	if (!tk.cnv) return
 	tk.legend.cnv.holder.selectAll('*').remove()
```

Comment:
thanks looks good.
i wonder if possible for `may_create_cnv()` to create persistent colorscale instance, then in may_update_cnv() will update the instance

Reply:
The change in [4a4806f](https://github.com/stjude/proteinpaint/pull/2252/commits/4a4806fbdceff3d04e2757d37a691d1888a6c9b9) should work. I don't scenario to test it though.